packages/victory-core/src/types/callbacks.ts

```diff
@@ -51,5 +51,3 @@ export type PaddingOrCallback = number | BlockProps | VictoryPaddingCallback;
 export type OrientationOrCallback =
   | OrientationTypes
   | VictoryOrientationCallback;
-
-export type ValueOrCallback<TValue> = TValue | ((args: CallbackArgs) => TValue);
```

Comment:
Looks like `types:check` is failing because of a couple instances where this is still being used.

Reply:
Oh weird, I don't know how I missed those.